applications/conversations/views/messages/popin.php

```diff
@@ -45,7 +45,7 @@
         <?php endforeach; ?>
         <li class="Item Center">
             <?php
-            echo anchor(sprintf(t('All %s'), t('Messages')), '/messages/inbox');
+            echo anchor(t('All Messages'), '/messages/inbox');
```

Comment:
I've added the source string with https://github.com/vanilla/locales/commit/d95e4ff74a585bbd721d678837eda47a5c4c86fe but we'll need to wait until it's been translated before this backport is merged.